### PR TITLE
Removed redundant

### PR DESCRIPTION
**Pull request by justgo129**

### CODE REVIEW

**Comment by justgo129** [comment]
(Sorry, I haven't figured out how to rebase in the UI; otherwise this would have been part of #144).

File: lib/Tuba/files/templates/doc/resources.html.ep [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <h2><center>Data Model</center></h2>
 <p>The GCIS maintains two concurrent models of the data : a relational model, and a semantic model.
 Entities represented by the GCIS are refered to as <b>resources</b>, as defined in <br /><a href="http://tools.ietf.org/html/rfc2396" target="_blank">RFC 2396</a>.
-They are assigned Uniform Resource Identifiers, or URIs.</p> <h3>Resources</h3>
+They are assigned Uniform Resource Identifiers, or URIs.</p> <h3>Riesources</h3>
```
Comment (bduggan) [inline lib/Tuba/files/templates/doc/resources.html.ep]: Typo?

**Comment by bduggan** [comment]
Not merging since it adds a typo.

File: lib/Tuba/files/templates/doc/resources.html.ep (hunk 1 above)
Comment (justgo129) [inline lib/Tuba/files/templates/doc/resources.html.ep]: dang-don't know why I corrected line.  Good catch.

**Pull request closed without merging**